### ISSUE TITLE
Extension crashes Firefox when trying to find ToS;DR for tosdr.org

### ISSUE THREAD

**Issue by jynnie** [tosdr/browser-extensions#27]
When I attempt to open the extension while on tosdr.org, Firefox crashes.

**Comment by michielbdejong** [comment]
Hm, which version of Firefox do you have?
I use Firefox 63 on Ubuntu, and I didn't see it crash, but I did see a spinnner that did not disappear like it  should:
![screenshot from 2018-12-10 09-51-34](https://user-images.githubusercontent.com/408412/49720981-79c85700-fc61-11e8-8ccf-0438939a9b12.png)

**Comment by jynnie** [comment]
I'm using Firefox 63.0.3 on Mac. I also get the spinner, but once I click off or close the pop-up, it instantly crashes.

**Comment by jynnie** [comment]
I also just updated to Firefox 64 to check if that fixed the issue, but it still crashes.

**Comment by oslego** [comment]
This is caused by invoking an invalid CSS selector due to the semicolon in the `ToS;DR` service name concatenated in the query selector [expressions used with jQuery in `popup.js`](https://github.com/tosdr/browser-extensions/blob/3c3d9d47a2db8b3e86c2c740851b9b4453515e24/src/popup/js/popup.js#L43)

I encounter this on the Chrome extension on this page: https://edit.tosdr.org/
It's likely the same issue with Firefox.

![Screen Shot 2019-06-08 at 14 50 27](https://user-images.githubusercontent.com/63899/59147695-0f6d5500-89ff-11e9-8a68-79d9e8540494.jpg)

--- 
As an example: try to run this query selector command in plain JS in any page to observe the error.

```javascript
document.querySelector('#popup-point-ToS;DR-4518')

// VM26:1 Uncaught DOMException: Failed to execute 'querySelector' on 'Document': '#popup-point-ToS;DR-4518' is not a valid selector.
//    at <anonymous>:1:10
```
--- 
The `getServiceDetails()` promise does not catch the error thrown by `tosdrPoint()`, so the popup document remains stuck in the "loading" state.

Perhaps the `serviceName` should be sanitized when generating the element ID & querying for it. Or maybe dropping the `serviceName` altogether from the element ID if there's no real need for that distinction in the `popup.html`. The `dataPoint.id` should suffice to serve as a unique ID that's unique per service and services shown one at a time in the popup, right?

**Comment by JustinBack** [comment]
Fixed in 89e56898271bc1ec6404d46b12f100a9ca636e77
